docs/opening-with-new-project-system.md

```diff
@@ -1,12 +1,51 @@
 # Opening with the new project system
 
+## Background
+
+The solution (.sln) file specifies which project system should be used to open each project. For example:
+
+```
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Library3", "Library3.csproj", "{ADFEAAF5-225C-4E13-8B65-77057AAC44B8}"
+EndProject
+```
+
+Here the "Library3.csproj" project should be opened with the project system designated by the "FAE04EC0-301F-11D3-BF4B-00C04F79EFBC" GUID.
```

Comment:
Users reading this might be interested in other project type GUIDs too. We don't have an official doc that lists these that I know of, but this source file does a decent job of enumerating the ones that are most likely interesting to the reader:

https://github.com/dotnet/project-system/blob/1aa6689827ba43e8cd7b9d29a6d15b3eabf6842c/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/ProjectType.cs